Modules/Core/Support/CustomValuesHelper.php

```
@@ -0,0 +1,231 @@
+     *
+     * @return string
+     */
+    public static function format_singlechoice($txt)
+    {
+        if ($txt == null) {
+            return '';
+        }
+    
+        // TODO: Migrate remaining CodeIgniter dependencies to Laravel
+        // TODO: Replace with Laravel equivalent: // load->model('custom_values/mdl_custom_values', 'cv');
+        $el = $CI->cv->get_by_id($txt)->row();
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$CI' which will lead to PHP notices. (undefined)

(UndefinedVariable)